### PR TITLE
tailscale: new, 1.16.2

### PR DESCRIPTION
**Pull request by RedL0tus**
Topic Description
-----------------

Introduction of Tailscale, a mesh VPN service based on the WireGuard protocol.

Package(s) Affected
-------------------

tailscale: new, 1.16.2

Security Update?
----------------

No

Test Build(s) Done
------------------

**Primary Architectures**

- [x] AMD64 `amd64`   
- [x] AArch64 `arm64`
    

**Secondary Architectures**

Architectural progress for "secondary," or experimental ports does not impede on merging of this topic.

- [x] Loongson 3 `loongson3`
- [x] PowerPC 64-bit (Little Endian) `ppc64el`

Update(s) Uploaded to Stable
----------------------------

**Primary Architectures**

- [x] AMD64 `amd64`   
- [x] AArch64 `arm64`

**Secondary Architectures**

Architectural progress for "secondary," or experimental ports does not impede on merging of this topic.

- [x] Loongson 3 `loongson3`
- [x] PowerPC 64-bit (Little Endian) `ppc64el`

### CODE REVIEW

File: extra-network/tailscale/autobuild/build [hunk 1]
```diff
@@ -0,0 +1,7 @@
+cd "${SRCDIR}"
```
Comment (MingcongBai) [inline extra-network/tailscale/autobuild/build]: This ain't no PKGBUILD, cowboy.

File: extra-network/tailscale/autobuild/build [hunk 2]
```diff
@@ -0,0 +1,7 @@
...
+abinfo "Building tailscale"
+"${SRCDIR}/build_dist.sh" tailscale.com/cmd/tailscale
+abinfo "Building tailscaled"
+"${SRCDIR}/build_dist.sh" tailscale.com/cmd/tailscaled
+abinfo "Installing binaries"
+install -Dvm755 tailscale tailscaled -t "${PKGDIR}/usr/bin"
```
Comment (MingcongBai) [inline extra-network/tailscale/autobuild/build]: Absolute path here.

**Review by MingcongBai — CHANGES_REQUESTED** [review]

**Review by MingcongBai — APPROVED** [review]

**Review by MingcongBai — CHANGES_REQUESTED** [review]
Waiting for tests results.

**Comment by RedL0tus** [comment]
> Waiting for tests results.

AArch64

AMD64

It werks senpai ! 🚀🚀🚀 @MingcongBai

**Review by MingcongBai — DISMISSED** [review]

**Comment by MingcongBai** [comment]
Fine, kid. Now get outta here.

**Review by WhiredPlanck — APPROVED** [review]

**Comment by RedL0tus** [comment]
1.16.2 also werks senpai tachi!  🚀🚀🚀

**Pull request merged**